src/rules/converters/variable-name.ts

```diff
@@ -0,0 +1,116 @@
+import { RuleConverter } from "../converter";
+
+export const convertVariableName: RuleConverter = tslintRule => {
+    const getCamelCaseRuleOptions = () => {
+        const camelCaseOptionNotice: string[] = [];
+        camelCaseOptionNotice.push(
+            "By default, this rule looks for any underscores (_) located within the source code. It ignores leading and trailing underscores and only checks those in the middle of a variable name.",
+        );
+
+        if (
+            tslintRule.ruleArguments.length !== 0 &&
```

Comment:
You are right, however I tend to check first the array length to avoid any extra lookup on the array itself. I don't know where I got that habit but nice to hear that suggestion :)